src/index.ts

```diff
@@ -98,7 +109,33 @@ class F1TelemetryClient extends EventEmitter {
    *
    * @param {Buffer} message
    */
-  parseMessage(message: Buffer) {
+  handleMessage(message: Buffer) {
+    if (this.socket && this.forwardPort) {
+      // forward message to port
+      this.socket.send(message, 0, message.length, this.forwardPort, '0.0.0.0');
```

Comment:
would also adding the ability to set IP address to forward to also be useful? Not all routers/home networks allow UDP broadcast, some throttle it.